src/Refraction/hughes.jl

```diff
@@ -84,14 +84,12 @@ function _refraction(model::HUGHES{T}, elevation_deg::T) where {T<:AbstractFloat
 
     # calculate refraction correction in arcseconds based on elevation angle
     if elevation_deg > T(5.0)
+        # TODO: use `evalpoly`/Horner's scheme instead!
         refract = T(58.1) / tan_el - T(0.07) / (tan_el^3) + T(8.6e-5) / (tan_el^5)
     elseif elevation_deg > T(-0.575)
-        refract =
-            elevation_deg * (
-                T(-518.2) +
-                elevation_deg *
-                (T(103.4) + elevation_deg * (T(-12.79) + elevation_deg * T(0.711)))
-            ) + T(1735.0)
+        refract = let p = (1735.0, -518.2, 103.4, -12.79, 0.711), q = map(T, p)
+            elevation_deg * evalpoly(elevation_deg, q)
```

Comment:
```suggestion
            evalpoly(elevation_deg, q)
```

This should not be multiplied by `elevation_deg`, that's why the test fails.